src/metabase/driver.clj

```diff
@@ -31,6 +32,8 @@
   `nil` (meaning subsequent queries will not attempt to change the session timezone) or something considered invalid
   by a given Database (meaning subsequent queries will fail to change the session timezone)."
   []
+  (when config/tests-available?
+    ((requiring-resolve 'mb.hawk.parallel/assert-test-is-not-parallel) `notify-all-databases-updated))
```

Comment:
We should not be doing this in parallel tests, for Snowflake it kills the open connection pool which will break other tests running at the same time.